### DISCUSSION TITLE
Here is what I did with this libary

### DISCUSSION THREAD

**Discussion by donnersm** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#149] [Show and tell]
Finally finished my setup using this amazing libary. 
Thanks you all for answering my questions and your support.
https://youtu.be/bQ7c9Vlhyp0

code and design:
https://github.com/donnersm/FFT_ESP32_Analyzer

**Comment by vortigont** [comment]
Great project indeed! I'll try to assemble it when time permit. Might contribute some ideas also! :)

**Comment by donnersm** [comment]
Forgot to say, here is the link to code and design:
https://github.com/donnersm/FFT_ESP32_Analyzer
